gpustat/core.py

```diff
@@ -372,22 +372,31 @@ def get_gpu_info(handle):
 
             def get_process_info(nv_process):
                 """Get the process information of specific pid"""
-                process = {}
                 if nv_process.pid not in GPUStatCollection.global_processes:
                     GPUStatCollection.global_processes[nv_process.pid] = \
                         psutil.Process(pid=nv_process.pid)
                 ps_process = GPUStatCollection.global_processes[nv_process.pid]
 
+                process = {
+                    'pid': ps_process.pid,
+                    'username': 'unknown',
+                    'command': '?',
+                    'full_command': ['?'],
+                    'gpu_memory_usage': None,
+                    'cpu_percent': None,
+                    'cpu_memory_usage': None,
+                }
                 # TODO: ps_process is being cached, but the dict below is not.
-                process['username'] = ps_process.username()
+                try:
+                    # On windows, not all process can be queried
+                    process['username'] = ps_process.username()
+                except psutil.AccessDenied:
+                    return process
```

Comment:
If we don't want to log these processes, just add `psutil.AccessDenied` to the except block here:

https://github.com/wookayin/gpustat/blob/847f87e531432b603c805f2289621e73881f653d/gpustat/core.py#L484-L497